package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@floip/flow-runner",
-  "version": "1.0.0-rc3",
+  "version": "1.0.0-rc4.1",
```

Comment:
Why 4.1 instead of just rc4?

Reply:
I created a version `1.0.0-rc4` when I though the flow-runner side is good. see this [commit](https://github.com/FLOIP/flow-runner/pull/68/commits/8c4f2bbe5fa515fb411b1d22b8d68c2854c6edf6) 
Then, I started to consume it from flow-builder and realized there were several changes I should have done.
Therefore, once the flow-builder is good, I created a new build on flow-runner side, and bumped the initial version to `1.0.0-rc4.1`. That's why the diff here is `rc4.1` instead `rc4`

Fyi: when we add new changes to rc4, we will bump that version again, in analogy of semantic versioning maintenance https://semver.org/#summary

Reply:
Thanks for the explanation.